books-call-number/nodes/buildQuery.json

```
@@ -0,0 +1,8 @@
+{
+  "id": "b78799bb-4e37-420b-bc06-eaf21ba8200e",
+  "name": "Build Query",
+  "description": "Build Query using input parameters - startRange and endRange.",
```

Can we update the description to not include required variables? It would be good to know what the required input variables for the task are but not sure we want to convolute the description. We do not want to maintain descriptions for when variables change for a given task. Additionally, the JavaScript query also requires `logLevel`. I would like to discuss where we should document and manage the process and local variables used within a workflow. Should they be a property on a persistent entity to be able to perform input checks once per workflow and not at every workflow. Such as the setup task iterate over the tasks and determine if any input/output variable is used or not and what scope.

---

The setup task can provide fail fast incident which can provides more details in the exception for preventing arbitrary task failure. It would be location for workflow server-side validation. Are all the require variables defined and correct type?

---

None of these are changes from the community sprint and these requests are out of scope.
Their respective PRs should have had these reviews.

That PR is here: https://github.com/TAMULib/fw-registry/pull/329